cfd_protocol/src/lib.rs

```diff
@@ -255,7 +399,7 @@ pub struct Payout {
 }
 
 impl Payout {
-    fn to_txouts(&self, maker_address: &Address, taker_address: &Address) -> Vec<TxOut> {
+    fn as_txouts(&self, maker_address: &Address, taker_address: &Address) -> Vec<TxOut> {
```

Comment:
`as_` should be a cheap conversion by convention. You are returning an owned instance from a reference (`&self`) which implies an allocation. I think this should be named `to_`.

Did clippy suggest this?

Reply:
I blame Clippy! (It was Clippy)